test-data/unit/daemon.test

```diff
@@ -29,15 +29,15 @@ def f(): pass
 $ dmypy run -- foo.py --follow-imports=error
 Daemon started
 $ dmypy run -- foo.py --follow-imports=error
-$ echo '[mypy]' >mypy.ini
-$ echo 'disallow_untyped_defs = True' >>mypy.ini
+$ {python} -c "print('[mypy]')" >mypy.ini
```

Comment:
Sigh.

Reply:
Yeah this is unfortunate :(

Sadly I couldn't find an echo call that made both shells happy.